smrf/framework/model_framework.py

```diff
@@ -620,18 +629,20 @@ def create_distributed_threads(self):
         self._logger.info("Initializing distributed variables...")
 
         # These are the variables that will be queued
-        thread_queue_variables = list(self.BASE_THREAD_VARIABLES)
```

Comment:
One of my thoughts here was to keep the `thread_queue_variables` to the local method scope and not have it set on the instance. That was a guess though on where it potentially could persist across runs.

Reply:
It seems that this probably isn't where things are being persisted across runs if it's still happening. I've set it on the class to try and see if it's changing in length. Might have to be a little more aggressive in the tests to see where it's failing.